### PR TITLE
add option for switching configuration by using environment variables

### PR DESCRIPTION
**Pull request by val-sytch**
by default it takes development configuration from config.ini
To switch mode 
1. export DJANGO_PROJ_MODE=_prod
return to origin settings:
1. unset DJANGO_PROJ_MODE

### CODE REVIEW

File: djog/config/config.py [hunk 1]
```diff
@@ -3,7 +3,11 @@
 
 path = os.path.realpath(os.path.dirname(__file__))
 config = configparser.ConfigParser(interpolation=None)
-config.read(os.path.join(path, 'config.ini'))
+
+if os.environ['DJANGO_PROJ_MODE'] == 'prod':
```
Comment (SpikePyth) [inline djog/config/config.py]: just read config file with env suffix. please don't do sux ugly `if`

File: manage.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "shop_django.settings")
+    # DJANGO_PROJ_MODE could be 'prod'-for production config or anything else for development config
+    os.environ['DJANGO_PROJ_MODE'] = 'dev'
```
Comment (SpikePyth) [inline manage.py:7]: set this as env variable 
`export DJANGO_PROJ_MODE=dev`

**Comment by val-sytch** [comment]
@SpikePyth better?

File: djog/config/config.py [hunk 3]
```diff
@@ -3,7 +3,11 @@
 
 path = os.path.realpath(os.path.dirname(__file__))
 config = configparser.ConfigParser(interpolation=None)
-config.read(os.path.join(path, 'config.ini'))
+try:
+    file_config = 'config{0}.ini'.format(os.environ['DJANGO_PROJ_MODE'])
```
Comment (SpikePyth) [inline djog/config/config.py]: `'config{0}.ini'.format("_" + os.environ.get('DJANGO_PROJ_MODE') if os.environ.get('DJANGO_PROJ_MODE') else '')`

Comment (val-sytch) [reply djog/config/config.py]: or even shorter without if
'config{0}.ini'.format(os.getenv('DJANGO_PROJ_MODE', ''))

**Pull request merged**